### PR TITLE
[newrelic-logging] Bump newrelic-fluentbit-output image to 2.0.0

### PR DESCRIPTION
**Pull request by nr-dsharma**
#### Which issue this PR fixes
It bumps the image for newrelic-logging integration, it uses the plugin version 2.0.0 that user fluent-bit 3.0.4 for docker images. https://github.com/newrelic/newrelic-fluent-bit-output/pull/154

it fixes critical vulnerability https://nvd.nist.gov/vuln/detail/CVE-2024-4323 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/newrelic-logging/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: A Helm chart to deploy New Relic Kubernetes Logging as a DaemonSet, supporting both Linux and Windows nodes and containers
 name: newrelic-logging
-version: 1.21.5
-appVersion: 1.19.2
+version: 1.21.6
```
Comment (luckslovez) [inline charts/newrelic-logging/Chart.yaml:4]: I'd be tempted to bump major here, at least minor, definetely not patch, wdyt?

Comment (nr-dsharma) [reply charts/newrelic-logging/Chart.yaml]: updated to minor version

**Review by danybmx — APPROVED** [review]
Kudos @nr-dsharma!

**Review by jsubirat — APPROVED** [review]
🚀

**Pull request merged**